src/python_testing/TC_AVSM_2_1.py

```diff
@@ -292,145 +315,174 @@ async def test_TC_AVSM_2_1(self):
                 endpoint=endpoint, cluster=cluster, attribute=attr.SoftRecordingPrivacyModeEnabled
             )
             logger.info(f"Rx'd SoftRecordingPrivacyModeEnabled: {value}")
+            asserts.assert_is_not_none(value, "SoftRecordingPrivacyModeEnabled is None")
 
         self.step(22)
         if self.pics_guard(self.check_pics("AVSM.S.A0014")):
             value = await self.read_single_attribute_check_success(
                 endpoint=endpoint, cluster=cluster, attribute=attr.SoftLivestreamPrivacyModeEnabled
             )
             logger.info(f"Rx'd SoftLivestreamPrivacyModeEnabled: {value}")
+            asserts.assert_is_not_none(value, "SoftLivestreamPrivacyModeEnabled is None")
 
         self.step(23)
         if self.pics_guard(self.check_pics("AVSM.S.A0015")):
             value = await self.read_single_attribute_check_success(
                 endpoint=endpoint, cluster=cluster, attribute=attr.HardPrivacyModeOn
             )
             logger.info(f"Rx'd HardPrivacyModeOn: {value}")
+            asserts.assert_is_not_none(value, "HardPrivacyModeOn is None")
 
         self.step(24)
         if self.pics_guard(self.check_pics("AVSM.S.A0016")):
             value = await self.read_single_attribute_check_success(endpoint=endpoint, cluster=cluster, attribute=attr.NightVision)
             logger.info(f"Rx'd NightVision: {value}")
+            asserts.assert_is_not_none(value, "NightVision is None")
```

Comment:
This also needs to test that the value is valid for a TriStateAutoEnum

Reply:
Thank you for review. We havd submitted it.